packages/models/opportunity.ts

```diff
@@ -15,6 +15,8 @@ export interface Opportunity {
     published_at: Date;
     created_at: Date;
     projects: { data: Project[] };
+    roleType: string;
+    roleCategory: string;
```

Comment:
@VictorDiniz89 this is why https://github.com/dev-launchers/strapiv4/pull/210 is very useful. We will be able to use the enum values defined in the backend here instead of string.